core/src/main/java/io/questdb/cutlass/text/CsvFileIndexer.java

```diff
@@ -548,7 +548,7 @@ private IndexOutputFile prepareTargetFile(long partitionKey) {
             }
         }
 
-        path.chop$().put(index);
+        path.put(index);
```

Comment:
why "chop" is removed ? the index will be added after 0

On a side note, i would avoid baking in so many code improvements in a PR that is doing something else.